package.json

```diff
@@ -20,12 +20,14 @@
   "homepage": "https://github.com/Vlad160/Simple-Components#readme",
   "dependencies": {
     "custom-elements": "^1.0.2",
-    "typescript": "^2.7.2",
-    "webpack": "^4.1.0"
+    "lodash": "^4.17.5",
+    "typescript": "^2.7.2"
```

Comment:
Can move this to devDependencies section

Reply:
Done